test/utils.test.js

```diff
@@ -8,7 +8,7 @@ describe('utils', function () {
   describe('ascii', function () {
     it('should return the correct string', function () {
       var string = 'Test String%!@#\n';
-      var buffer = new Buffer(string);
+      var buffer = Buffer.from(string);
```

Comment:
Updated this because `new Buffer(...)` is deprecated.

https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe